docs/development-guide.md

```diff
@@ -257,3 +257,40 @@ The flow for getting a fix into a release branch is:
 1. If there are any conflicts you will need to resolve them by pulling the
    branch, resolving the conflicts and force push back the branch
 1. After the CI is green, the bot will automatically merge the backport PR.
+
+### Retriggering the CI Jobs
+
+The CI Jobs gets triggered automatically on these events, such as on
+opening fresh PRs, rebase of PRs and force pushing changes to existing PRs.
+
+Right now, we also have below commands to manually retrigger the CI jobs
+
+1. To retrigger all the CI jobs, comment the PR with command: `/retest all`
+
+   **Note**:
+
+   * This will rerun all the jobs including the jobs which are already passed
```

Comment:
This indention is an issue. Maybe don't make this a list, and just add it on the line after the **Note**.

Reply:
ok. My bad got lazy to run markdown lint locally. Let me run it this time.

Reply:
It passed locally this time :-)